### ISSUE TITLE
false positive for unused kapt processors when the processor is in a non-kapt configuration dependency

### ISSUE THREAD

**Issue by RBusarow** [RBusarow/ModuleCheck#347]
Given the build file:

```kotlin
plugins {
  kotlin("jvm")
  kotlin("kapt")
}

dependencies {
  implementation("com.google.dagger:dagger-compiler:2.40.5")

  kapt(project(:"some-processor"))
}
```

ModuleCheck will say that the Dagger compiler dependency is an unused kapt processor and should be removed.

The rule should only check for dependencies from a kapt configuration.
